loadtest/tests.go

```diff
@@ -248,9 +248,16 @@ func actionGetChannel(c *EntityConfig) {
 		cmdlog.Errorf("Unable to get channel stats. Channel: %v, User: %v, Error: %v", channelId, c.UserData.Username, resp.Error.Error())
 	}
 
-	if posts, resp := c.Client.GetPostsForChannel(channelId, 0, 60, ""); resp.Error != nil {
+	if posts, resp := c.Client.GetPostsForChannel(channelId, 0, 60, c.Etags.GetPostsForChannel); resp.Error != nil {
```

Comment:
Just to confirm, does an entity typically browse more than one channel, and are we consciously simulating using the etag from a previous channel request for all subsequent ones, even if the channel changes? Would it be easy to just keep track of a map of etags by channel id if I've understood the changes thus far?

Reply:
Yeah that would make more use of the etag for sure, I was mainly just trying to get any usage of it and figured it would better to submit a PR with what I had than just throw it away